django_statsd/clients/__init__.py

```diff
@@ -1,14 +1,31 @@
 from __future__ import absolute_import
+from datetime import datetime, timedelta
 from importlib import import_module
+
 from django.conf import settings
 
 
-def get_client():
-    client = getattr(settings, 'STATSD_CLIENT', 'statsd.client')
-    host = getattr(settings, 'STATSD_HOST', 'localhost')
-    port = getattr(settings, 'STATSD_PORT', 8125)
-    prefix = getattr(settings, 'STATSD_PREFIX', None)
-    return import_module(client).StatsClient(host=host, port=port, prefix=prefix)
+class StatsdClientProxy:
```

Comment:
I looked in the repo and saw there are already some unit tests so I think we should also add two tests for this code
1. that confirms that the underlying client is called
2. that a new client is initialized after a given time (setting can be overridden)

Reply:
Good catch about the tests. In fact, I needed to make some changes to make them pass anyway.

The fact that the underlying client is called is already covered by the existing tests. I added a test for the re-initializing.

The tests aren't run automatically, but I ran them locally. You can do the same if you want :)